multimap/src/main/java/org/infinispan/multimap/impl/EmbeddedSetCache.java

```diff
@@ -163,15 +165,19 @@ public CompletionStage<Collection<V>> pop(K key, Long count, boolean remove) {
    }
 
    /**
-    * Returns if a set contains an element
+    * Returns a list L of 0/1. L(i) = 1 if the set contains elements(i)
     *
-    * @param key, the name of the set
-    * @param element, the element
-    * @return {@link CompletionStage} containing a boolean
+    * @param key,      the name of the set
+    * @param elements, list of elements
+    * @return {@link CompletionStage} containing a List<Long> or null if elements
+    *         null or empty
     */
-   public CompletionStage<Boolean> contains(K key, V element) {
+   public CompletionStage<List<Long>> mIsMember(K key, V... elements) {
       requireNonNull(key, ERR_KEY_CAN_T_BE_NULL);
-      return cache.getAsync(key).thenApply(b -> b == null ? false : b.contains(element));
+      if (elements == null || elements.length == 0) {
+         return CompletableFuture.completedStage(null);
```

Comment:
On CI:

```
07:12:35  [ERROR] /home/infinispan/workspace/Infinispan_PR-12623/multimap/src/main/java/org/infinispan/multimap/impl/EmbeddedSetCache.java:178:49: [not required for tests] Forbidden method invocation found that matches (completed|failed)Stage [ForbiddenMethod]
```

You can return `CompletableFutures.completedNull()` instead.

Reply:
ouch!